docs/MS Word/Оглавление.md

```diff
@@ -0,0 +1,48 @@
+﻿# Оглавление в MS Word: Руководство по вставке, обновлению и стилям
+## Введение
+Microsoft Word — один из наиболее распространенных текстовых редакторов,
+который предоставляет обширные возможности для оформления документов.
+Одной из важных частей форматирования является оглавление.
+В этой статье мы рассмотрим, как вставлять оглавление, обновлять его,
+а также применять стили для лучшего визуального восприятия.
+## Вставка оглавления
+Для вставки оглавления в MS Word, выполните следующие шаги:
+
+1. **Выберите место для оглавления:**
```

Comment:
[ch-md-warning-emphasis-punctuation](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-warning-emphasis-punctuation)